src/psyclone/tests/generator_test.py

```diff
@@ -567,14 +567,32 @@ def test_main_unexpected_fatal_error(capsys, monkeypatch):
     assert expected_output in output
 
 
-def test_main_fort_line_length(capsys):
+@pytest.mark.parametrize("limit", ['all', 'output'])
```

Comment:
Add a test to check -l off works as expected, as this does not appear to be checked anywhere.

Reply:
Oooh, that was embarrassing - it turns out I'd messed up and line-length limiting was on by default. Fixed now.

Reply:
:-). Tests are great.